sound/soc/sof/ipc.c

```diff
@@ -1025,6 +1025,9 @@ struct snd_sof_ipc *snd_sof_ipc_init(struct snd_sof_dev *sdev)
 
 	init_waitqueue_head(&msg->waitq);
 
+	/* set IPC ops. IPC3 by default */
+	ipc->ops = &ipc3_ops;
```

Comment:
This is probably a platform-specific ops? Or is the plan to do this later?

Reply:
right. That would be based on the IPC selection logic which we don't have yet

Reply:
0k, let's park this comment for now.